app_base.css

```diff
@@ -0,0 +1,34 @@
+#category_list{
+  list-style-type: none;
+  padding-left: 0px;
+  font-family: 'Karma', serif;
+}
+
+.activeone{
+  font-weight: bold;
+  font-size: 20px;
+  padding: 3px 6px;
+}
+
+a{
+  color: black;
+}
+
+.five_history {
+  list-style-type: none;
+  font-family: 'Karma', serif;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.five_history ol li{
```

Comment:
I didn't know that you could do this! I did something like .five_history > ol > li

Reply:
I love not using the annoying greater than and less than brackets in the css. Not sure which is more correct, but I always like the idea of LESS IS MORE.

Reply:
I'm just a CSS wizard